### ISSUE TITLE
"args.map is not iterable"

### ISSUE THREAD

**Issue by jherdman** [ember-cli/ember-cli-update#1096]
From today:

```
TypeError: args.map is not a function or its return value is not iterable
    at spawn (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/run.js:22:22)
    at Module.npm [as exports] (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:6:16)
    at AsyncFunction.npmJson [as json] (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:10:45)
    at Object.getVersions (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/src/get-versions.js:15:20)
    at /home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/src/check-for-blueprint-updates.js:18:34
    at Array.map (<anonymous>)
    at checkForBlueprintUpdates (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/src/check-for-blueprint-updates.js:13:39)
    at chooseBlueprintUpdates (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/src/choose-blueprint-updates.js:86:34)
    at emberCliUpdate (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/src/index.js:132:15)
    at async Object.handler (/home/james/.npm/_npx/636224/lib/node_modules/ember-cli-update/bin/commands/default.js:32:16)
```

## Debug info

Node v12.15.0

(NOTE: the below snippet is exactly the same as above)

```
[nix-shell:~/DEV/PN/date-picker](update-ember-and-friends)$ DEBUG=ember-cli-update,boilerplate-update,git-diff-apply ember-cli-update
TypeError: args.map is not a function or its return value is not iterable
    at spawn (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/run.js:22:22)
    at Module.npm [as exports] (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:6:16)
    at AsyncFunction.npmJson [as json] (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:10:45)
    at Object.getVersions (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/src/get-versions.js:15:20)
    at /home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/src/check-for-blueprint-updates.js:18:34
    at Array.map (<anonymous>)
    at checkForBlueprintUpdates (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/src/check-for-blueprint-updates.js:13:39)
    at chooseBlueprintUpdates (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/src/choose-blueprint-updates.js:86:34)
    at emberCliUpdate (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/src/index.js:132:15)
    at async Object.handler (/home/james/DEV/PN/date-picker/.nix-node/lib/node_modules/ember-cli-update/bin/commands/default.js:32:16)
```

**Comment by jherdman** [comment]
I tried updating to Node v14 and saw the same error.

**Comment by kellyselden** [comment]
Sorry about that. Try now.

**Comment by michael-lowe-nz** [comment]
I'm also getting the same error:

```
TypeError: args.map is not a function or its return value is not iterable
    at spawn (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/run.js:22:22)
    at Module.npm [as exports] (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:6:16)
    at AsyncFunction.npmJson [as json] (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:10:45)
    at Object.getVersions (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/src/get-versions.js:15:20)
    at loadDefaultBlueprintFromDisk (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/src/load-default-blueprint-from-disk.js:37:34)
    at async emberCliUpdate (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/src/index.js:145:17)
    at async Object.handler (/Users/michaellowe/.nvm/versions/node/v12.8.1/lib/node_modules/ember-cli-update/bin/commands/default.js:32:16)
```

```
ember-cli: 2.18.2
node: 12.8.1
os: darwin x64
```

**Comment by progand** [comment]
Version
```
$ ember-cli-update --version
0.55.0
```
```
$ ember --version
ember-cli: 2.18.2
node: 12.20.0
os: linux x64
```

Error:
```
$ ember-cli-update --stats-only
TypeError: args.map is not a function or its return value is not iterable
    at spawn (/usr/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/run.js:22:22)
    at Module.npm [as exports] (/usr/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:6:16)
    at AsyncFunction.npmJson [as json] (/usr/lib/node_modules/ember-cli-update/node_modules/boilerplate-update/src/npm.js:10:45)
    at Object.getVersions (/usr/lib/node_modules/ember-cli-update/src/get-versions.js:15:20)
    at loadDefaultBlueprintFromDisk (/usr/lib/node_modules/ember-cli-update/src/load-default-blueprint-from-disk.js:37:34)
    at async stats (/usr/lib/node_modules/ember-cli-update/src/stats.js:36:17)
    at async Object.handler (/usr/lib/node_modules/ember-cli-update/bin/commands/default.js:42:19)
```

**Comment by kellyselden** [comment]
Can you try v0.55.1?

**Comment by MichalBryxi** [comment]
Started working for me again. The issue with `args.map is not iterable` went away when using:

```
❯ ember --version
dember-cli: 3.23.0
node: 12.20.0
os: darwin x64

❯ ember-cli-update --version
0.55.1
```

**Comment by jherdman** [comment]
Works! Just a note that I'm seeing the following warning though:

```
fixturify.readSync no longer supports options.include, please use options.globs instead
```

**Comment by kellyselden** [comment]
Thanks. That issue also should be fixed in v0.55.2.
